docusaurus/docs/Android/04-compose/04-message-components/06-selected-message-overlay.mdx

```diff
@@ -98,3 +102,71 @@ fun SelectedMessageOverlay(
 * `messageOptions`: Allows you to customize which message options are shown in the overlay. You can use `defaultMessageOptions()` to get the default actions that we expose in our SDK.
 
 If you want to customize these options or reactions, you can set up custom reactions, using the [`ChatTheme`](../02-general-customization/01-chat-theme.mdx)'s `reactionTypes` parameter.
+
+By default `SelectedMessageMenu` is made to resemble a bottom sheet, however it can be stylised in order to look like a completely different component.
+
+```kotlin
+SelectedMessageMenu(
+    // Use a Modifier to customize the appearance
+    modifier = Modifier
+        .align(Alignment.Center)
+        .padding(horizontal = 20.dp)
+        .wrapContentSize(),
+    // Assign a different shape to the Composable element
+    shape = ChatTheme.shapes.attachment,
+    messageOptions = defaultMessageOptionsState(selectedMessage = selectedMessage,
+        currentUser = user,
+        isInThread = listViewModel.isInThread),
+    message = selectedMessage,
+    onMessageAction = { action ->
+        // Handle message action
+    },
+    onDismiss = {
+        // Handle dismiss
+    }
+)
+```
+
+The code above will produce the following UI:
+
+![Stylised SelectedMessageMenu component](../../assets/compose_custom_selected_message_menu_shape_and_alignment.png)
+
+`SelectedMessageMenu` provides you with Composable slots that are ready for more extensive customisations.
+
+```kotlin
+    public fun SelectedMessageMenu(
+        ...,
+        headerContent: @Composable ColumnScope.() -> Unit = {
+        // Header content
+        },
+        bodyContent: @Composable ColumnScope.() -> Unit = {
+        // Body content
+        }
+    )
+```
+
+By default the `headerContent` slot contains `ReactionOptions` while the `bodyContent` slot contains `MessageOptions`. If you decide you like the behavior that `SelectedMessageMenu` provides, but do not need it's default content you can easily override one or both of it's content slots.
+
+```kotlin
+if (selectedMessage != null) {
+        ...,
```

Comment:
This is kind of confusing - we should still show which component we're using here.